### ISSUE TITLE
SMTP issues with Google Apps

### ISSUE THREAD

**Issue by nsina** [InvoicePlane/InvoicePlane#77]
I have issues with the SMTP settings. 

It doesn't connect to the Google Apps SMTP.

```php
A PHP Error was encountered
Severity: Warning
Message: Cannot modify header information - headers already sent by (output started at XXX/InvoicePlane/application/modules/mailer/helpers/phpmailer/class.phpmailer.php:586)
Filename: libraries/Session.php
Line Number: 688

A PHP Error was encountered
Severity: Warning
Message: Cannot modify header information - headers already sent by (output started at XXX/InvoicePlane/application/modules/mailer/helpers/phpmailer/class.phpmailer.php:586)
Filename: helpers/url_helper.php
Line Number: 542
```

It seemed to work previously, but I can't recall exactly since I have installed several instances of InvoicePlane on my server. I am using the same emails as previously, so it can't be the email settings. 

Is this relating to permission issues?

**Comment by nsina** [comment]
After running some tests, I concluded that it wasn't InvoicePlane's fault that I got these errors.

**I contacted my web hosting provider and found out that they were blocking the outgoing ports for Google Apps SMTP, which resulted in these error codes when trying to send emails from InvoicePlane.** Issue solved after opening the ports.

I guess this issue is solved and we could probably close the thread, unless someone else has the same issue and hasn't been successful in solving it with their web host.
